packages/nodes-base/nodes/BambooHR/v1/actions/versionDescription.ts

```diff
@@ -0,0 +1,72 @@
+import {
+	INodeTypeDescription,
+} from 'n8n-workflow';
+import * as employees from './employees';
+import * as employeeFile from './employeeFile';
+import * as companyFile from './companyFile';
+import * as reports from './reports';
+import * as tabularData from './tabularData';
+import * as timeOff from './timeOff';
+
+export const versionDescription: INodeTypeDescription = {
+	displayName: 'BambooHR',
+	name: 'bambooHR',
+	icon: 'file:bambooHR.png',
```

Comment:
If available, SVGs are preferred.

Reply:
Only the old logo was present in svg format. :/